mods/creative/inventory.lua

```diff
@@ -1,5 +1,10 @@
 local player_inventory = {}
 
+-- Creates a blank detached inventory to be filled with contents later
```

Comment:
I need to investigate this. I personally think it's worth autogenerating game_api.txt from Luadoc/Doxygen